exporters/zipkin/src/main/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporterBuilder.java

```diff
@@ -26,11 +26,12 @@ public final class ZipkinSpanExporterBuilder {
   private Supplier<InetAddress> localIpAddressSupplier = LocalInetAddressSupplier.getInstance();
   @Nullable private Sender sender;
   private String endpoint = ZipkinSpanExporter.DEFAULT_ENDPOINT;
+  private boolean compressionEnabled = true;
```

Comment:
Let's leave the default as false for the purposes of this PR. We can revisit in a separate discussion whether this is acceptable and backwards compatible.

Reply:
Will do. Just FYI: Unless you set your own sender, the created `OkHttpSender` has `compressionEnabled = true` as default, so the behavior will change then. But I agree that it would be more consistent if all exporters would have the same default (also for the spec change).

Reply:
Done!

Reply:
>  the created OkHttpSender has compressionEnabled = true as default

I didn't realize that. My goal with the comment was to keep this PR separate from any default behavioral changes, so apologies for the flip flop but I think we should revert back to what you had (i.e. `private boolean compressionEnabled = false;`

Reply:
No worries. I've reverted the code and added a clarifying comment to the `compressionEnabled` field, explaining why it's enabled by default for Zipkin.